Runtime/Utils/WebRequestDispatcher.cs

```diff
@@ -31,6 +31,7 @@ public class WebRequestDispatcher
             Dictionary<string, string> headers = null,
             string payload = null,
             DownloadHandler downloadHandler = default,
+            bool sendAppIdInHeader = true,
```

Comment:
It's a quick way to solve the problem, but I'm not a fan of how we've now got a param specific to one RPM-bespoke header in the params for all requests that get made in the SDK, even if it is optional. It adds extra confusion where, one day, a partner will look at this and go 'what the hell is an app id header' because this is just a generic WebRequestDispatcher and now it's got some non generic web request logic in it...

Would you be open to considering another approach? Maybe you could wrap this class in ReadyPlayerMeWebRequestDispatcher which handles the logic of passing down the AppID header in the already existing 'headers' param of this class. Then you'd use the ReadyPlayerMeWebRequestDispatcher across the project, but the image downloading would use the base WebRequestDispatcher? You'd then have better separation of concerns but achieve the same goal.

Reply:
I came up with the different and much more simpler approach. Please have a look.